docusaurus.config.js

```diff
@@ -4,6 +4,16 @@ const lightCodeTheme = require('prism-react-renderer/themes/github');
 const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 const math = require('remark-math');
 const katex = require('rehype-katex');
```

Comment:
Does `rehype-katex` need to be removed?

Reply:
Of course, you can remove the dependencies of rehype-katex by the way.

Reply:
done.